app/component/itinerary/BoardingInformation.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import cx from 'classnames';
+import PlatformNumber from '../PlatformNumber';
+import { modeUsesTrack } from '../../util/modeUtils';
+import { isPlatformChanged } from '../../util/legUtils';
+
+/**
+ * BoardingInformation displays platform or track information for a departure leg.
```

Comment:
This introduces a confusing concept 'departure leg'. All legs depart. Could you document it as 'transit leg and change the prop from 'departure' to 'leg'?